frontend/src/components/navbar/NavbarSearchPanel.vue

```diff
@@ -130,6 +119,17 @@ export default {
 
         @media screen and (max-width: 1087px) {
             width: 88%;
+
+            .icon {
+                position: relative;
+                left: -28px;
+            }
+            
+            &:before {
```

Comment:
In my opinion, you can just add special css class with `display:none` for ` <span class="search-btn">Search</span>` by default, and then using @media classes, rewrite display prop